src/App.js

```diff
@@ -1,12 +1,66 @@
-import React from 'react';
+import React, { useState, useEffect} from 'react';
 import './App.css';
+import axios from "axios";
+import Button from "./components/Button/Button";
+import PokemonCard from "./components/PokemonCard/PokemonCard";
+import logo from './assets/logo.png'
+
 
 function App() {
-  return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
-  );
+    const [pokemons, setPokemons] = useState([]);
```

Comment:
Als `pokemons` nooit een array zal zijn, dan zou ik dat ook niet als default waarde nemen. Het is wat zuiverder als je kiest voor `undefined`
```suggestion
    const [pokemons, setPokemons] = useState();
```